core/src/main/scala/wartremover/contrib/warts/UnintendedLaziness.scala

```diff
@@ -0,0 +1,39 @@
+package org.wartremover
+package contrib.warts
+
+object UnintendedLaziness extends WartTraverser {
+  val (errorForFilterKeys: String, errorForMapValues: String) = {
+    def error(name: String) =
+      s"""GenMapLike#$name is disabled because it implicitly creates lazily evaluated collections.
+         |To create lazy collections, use the explicit view or toStream methods""".stripMargin
+
+    (error("filterKeys"), error("mapValues"))
+  }
+
+  def apply(u: WartUniverse): u.Traverser = {
+    import u.universe._
+
+    val genMapLikeSymbol = rootMirror.staticClass("scala.collection.GenMapLike")
```

Comment:
https://travis-ci.org/wartremover/wartremover-contrib/jobs/426244307#L1491

```
[info] [error] ## Exception when compiling 1 sources to /tmp/sbt_190ab5cb/basic/target/scala-2.13.0-M5/classes
[info] [error] class scala.collection.GenMapLike in compiler mirror not found.
[info] [error] scala.reflect.internal.Mirrors$RootsBase.staticClass(Mirrors.scala:120)
[info] [error] scala.reflect.internal.Mirrors$RootsBase.staticClass(Mirrors.scala:22)
[info] [error] org.wartremover.contrib.warts.UnintendedLaziness$.apply(UnintendedLaziness.scala:16)
[info] [error] org.wartremover.Plugin$Traverser$$anon$1.$anonfun$apply$2(Plugin.scala:74)
[info] [error] org.wartremover.Plugin$Traverser$$anon$1.$anonfun$apply$2$adapted(Plugin.scala:74)
[info] [error] scala.collection.immutable.List.foreach(List.scala:302)
[info] [error] org.wartremover.Plugin$Traverser$$anon$1.go$1(Plugin.scala:74)
[info] [error] org.wartremover.Plugin$Traverser$$anon$1.apply(Plugin.scala:76)
[info] [error] scala.tools.nsc.Global$GlobalPhase.applyPhase(Global.scala:437)
```

Reply:
Yes. The Wart should not be used when compiling to the 2.13 target. I'll try to make it no-op in 2.13 instead, in case anyone uses it accidentally (or needs to cross-compile).